tools/vscode_sql_extraction/src/sql_extraction_runner.ts

```diff
@@ -41,6 +41,7 @@ export class SqlExtractionRunner {
 
     return new Promise<Query[]>((resolve, reject) => {
       let json = '';
+      let errMsg = '';
       const process = execFile(this.execPath, args)
```

Comment:
There might be a way to disable buffering of stderr when you start a process. If so, implement that in another pull request.

Reply:
Hmm yeah I'll look into it.